integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -111,6 +111,7 @@ public void startTrinoContainer(
             .withExtraHosts(
                 ImmutableMap.<String, String>builder()
                     .put("host.docker.internal", "host-gateway")
+                    .put("gravitino-ci-hive", hiveContainerIp)
```

Comment:
I suggest using the constant HiveContainer.HOST_NAME in the HiveContainer class, and also in line 78.

Reply:
@diqiu50 Thanks for the suggestion! Do you mean by add the following method to HiveContainer class and use `.put(hiveContainer.getHostName(), hiveContainerIp)`?
``` java
public String getHostName() {
  return HOST_NAME;
}
```

Reply:
hi @zivali Thank you for your fixed this issue.
I think you can directly use `.put(HiveContainer.HOST_NAME, hiveContainerIp)`

Reply:
Thanks for the suggestion! Sorry I missed that.
I updated this PR. Could you help take a look again? Thank you.